### PR TITLE
DirectStorage bindings

### PR DESCRIPTION
**Pull request by Perksey**
pending dogfooding & self review

### CODE REVIEW

**Comment by Perksey** [comment]
This is ready for review and the bindings have received sufficient testing.

File: src/Core/Silk.NET.BuildTools/Cpp/Clang.cs [hunk 1]
```diff
@@ -214,7 +198,10 @@ static void WriteDiagnosticsThrowOnError(TranslationUnit unit)
             Console.WriteLine("Visting declarations...");
             VisitDecls(DeclsOf(translationUnitDecl, translationUnitDecl));
             // ReSharper restore BitwiseOperatorOnEnumWithoutFlags
-
+            if (traversals.Any(x => x.Contains("winnt.h")))
+            {
+                Debugger.Break();
+            }
```
Comment (Perksey) [inline src/Core/Silk.NET.BuildTools/Cpp/Clang.cs:204]:
```suggestion
```

**Comment by Perksey** [comment]
This is ready for review.

**Review by HurricanKai — APPROVED** [review]

**Comment by Perksey** [comment]
Uh... merge button is broken for me.

**Pull request merged**